ci/gpu/build.sh

```diff
@@ -43,11 +43,11 @@ python --version
 conda config --set ssl_verify False
 
 logger "conda install required packages"
-conda install -y -c conda-forge -c rapidsai -c rapidsai-nightly -c nvidia \
+conda install -y -c nvidia -c rapidsai -c rapidsai-nightly -c conda-forge -c defaults \
```

Comment:
I think you should be able to remove the channels from this command as they're controlled within CI now.